internal/service/outposts/outpost_assets_data_source.go

```diff
@@ -25,6 +28,31 @@ func DataSourceOutpostAssets() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"host_id_filter": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					ValidateFunc: validation.All(
+						validation.StringLenBetween(1, 50),
+						validation.StringMatch(regexp.MustCompile(`^^[A-Za-z0-9-]*$`), "must match [a-zA-Z0-9-]"),
+					),
+				},
+			},
+			"status_id_filter": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 2,
```

Comment:
The SDK mentions a minimum number of 1 for this attribute, recommend adding that here: https://docs.aws.amazon.com/outposts/latest/APIReference/API_ListAssets.html#API_ListAssets_RequestSyntax